### DISCUSSION TITLE
Support mass selection of files

### DISCUSSION THREAD

**Discussion by raluaces** [diced/zipline#438] [Ideas]
Support mass selection of files for tasks like deleting several at once.

**Comment by smashedr** [comment]
I think the File Gallery page should have the option to show in Gallery or List view. And in List View users should be able to individually select items, or select all; to perform operations such as delete.

**Comment by diced** [comment]
Will be added in v4, only through the table view as you can use the select checkboxes.

**Comment by diced** [comment]
should've been closed a while ago but this is not supported in v4.
